E2E-demo/example.env

```diff
@@ -3,8 +3,9 @@ PROJECT=<insert-project-name>
 TOKEN=<insert-token>
 USER_NAME=<insert-user-name>
```

Comment:
can we annotate that token/username are for openshift cluster?
also, annotate that "project" is "openshift namespace"